lld/test/ELF/linkerscript/data-segment-relro-ppc64.test

```diff
@@ -0,0 +1,79 @@
+# REQUIRES: x86
```

Comment:
Should this be ppc? It is causing failures on bots that don't build PPC such as the PS bots:
https://lab.llvm.org/buildbot/#/builders/139/builds/49724
https://lab.llvm.org/buildbot/#/builders/216/builds/27302

Reply:
I have made a change to the REQUIRES line to be `ppc` instead in 72bbac4.